src/main.rs

```diff
@@ -53,6 +53,26 @@ enum CommandLineArgs {
         #[structopt(required = true)]
         group_ids: Vec<String>,
     },
+    /// Adds the `new_members` to the  `group_id`. Note: requires the user's DeviceContext in the file "<user_id>.json".
+    GroupAddMembers {
+        /// UserId to use for adding members
+        #[structopt(short, long = "user")]
+        user_id: String,
+
+        /// group to add members to.
+        #[structopt(required = true)]
+        group_id: String,
```

Comment:
I _think_ this should be behind a flag instead of being an argument? Feels a bit weird now.